test/test_method.py

```diff
@@ -19,8 +19,10 @@
 from .method.testnuclear import *
 from .method.testorbitals import *
 from .method.testpopulation import *
-if sys.version_info[0] == 2:
-    from .method.testvolume import *
+# Test for volume is commented off at the moment due to changes in pyquante bridge.
```

Comment:
Could you rephrase this so that it's imperative and clearly a TODO?

Reply:
I've clarified this comment. Thanks